src/components/Certify/Certify.vue

```diff
@@ -4,7 +4,7 @@
       <v-container>
         <v-row class="pl-4" no-gutters>
           <v-col :cols="firstColumn" class="px-0">
-            <label><strong>Legal Name</strong></label>
+            <label :class="{'error-text': invalidSection}"><strong>Legal Name</strong></label>
```

Comment:
Does the class exist for this component?

Reply:
No, this is a global class. Native to our UI.
I wasn't sure if i should add it to the assets of the Shared repo..

Reply:
I see the shared components import the "local" theme (local to this repo or local to the app they're used in).

If this class doesn't exist in this repo, please add it.

Reply:
K, will do 👍

Reply:
Fixed!